Sources/Citadel/Server.swift

```diff
@@ -1,10 +1,7 @@
 import NIO
+import Logging
 import NIOSSH
 
-public protocol SSHServerDelegate: NIOSSHServerUserAuthenticationDelegate, GlobalRequestDelegate {
```

Comment:
This is public api, don't we need to keep this/ why does it need to go? (I love cleaning up though)

Reply:
It was serving no purpose yet